python/smashbox/utilities/__init__.py

```diff
@@ -321,6 +321,11 @@ def expect_webdav_exist(path, user_num=None):
     exists = stdout.rstrip() == "0"
     error_check(exists, "Remote path %s exists but should not" % path)
 
+def expect_webdav_isfile(path, user_num=None):
```

Comment:
we can't change that file :(

Move to your test file for now.
You can also send a PR to cernbox and once they merged it we can merge it here again.